lib/3scale_toolbox/commands/remote_command/remote_add.rb

```diff
@@ -17,7 +17,36 @@ def self.command
         end
 
         def run
-          puts "remote add"
+          validate_input_params
+          begin
+            add_remote(*arguments[0..1])
+          rescue StandardError => e
+            warn e.message
+            exit 1
+          end
+        end
+
+        def validate_input_params
+          return unless arguments.size != 2
```

Comment:
This is not done by the CLI library? You can't require number of arguments ? It still executes the command?

Reply:
not sure, will check.

Reply:
done by  CLI library